### ISSUE TITLE
Internal error: Argument 5 passed to PHPStan\Analyser\Scope::enterClassMethod() must be of the type string or null, boolean given

### ISSUE THREAD

**Issue by ddebin** [phpstan/phpstan-doctrine#63]
phpstan/phpstan#2148

```
PHP Fatal error:  Uncaught TypeError: Argument 5 passed to PHPStan\Analyser\Scope::enterClassMethod() must be of the type string or null, boolean given, called in vendor/phpstan/phpstan-doctrine/src/Type/Doctrine/QueryBuilder/QueryBuilderMethodDynamicReturnTypeExtension.php on line 180 and defined in phar://vendor/phpstan/phpstan-shim/phpstan.phar/src/Analyser/Scope.php:1265
Stack trace:
#0 vendor/phpstan/phpstan-doctrine/src/Type/Doctrine/QueryBuilder/QueryBuilderMethodDynamicReturnTypeExtension.php(180): PHPStan\Analyser\Scope->enterClassMethod(Object(PhpParser\Node\Stmt\ClassMethod), Array, NULL, NULL, false, false, false)
#1 vendor/phpstan/phpstan-doctrine/src/Type/Doctrine/QueryBuilder/QueryBuilderMethodDynamicReturnTypeExtension.php(108): PHPStan\Type\Doctrine\QueryBuilder\QueryBuilderMethodDynamicReturnTypeExtension->findQueryBuilderTypesInCalledMetho in phar://vendor/phpstan/phpstan-shim/phpstan.phar/src/Analyser/Scope.php on line 1265` 
```

**Comment by oallain** [comment]
I fixed it in https://github.com/phpstan/phpstan-doctrine/pull/64

**Comment by ondrejmirtes** [comment]
Released as 0.11.5, thank you!
